lib/ClusterShell/Worker/Ssh.py

```diff
@@ -57,7 +57,8 @@ def _build_cmd(self):
             cmd_l += [os.path.expanduser(opt) for opt in shlex.split(options)]
 
         # Hardwired options (overridable by ssh_options)
-        cmd_l += [  "-a", "-x"  ]
+        # note: you should use only long-fromat options here
+        cmd_l += [  "-oForwardAgent=no", "-oForwardX11=no"  ]
```

Comment:
(style) could you remove the useless whitespace (PEP8) around brackets : ["-oFor...", "-oFor..."]